res/values/strings.xml

```diff
@@ -26,6 +26,7 @@
     <string name="unarchive">Unarchive</string>
     <string name="mute">Mute</string>
     <string name="ephemeral_messages">Disappearing messages</string>
+    <string name="ephemeral_messages_hint">These settings apply to all participants using Delta Chat. However, you must trust your conversation partners to use Delta Chat and not copy, save or forward any content.</string>
```

Comment:
This does not sound good when translated. How about "However, they may copy, save and forward messages or use other chat clients."

Reply:
sounds good to me!